src/skydb.ts

```diff
@@ -2,15 +2,28 @@ import { pki } from "node-forge";
 import { SkynetClient } from "./client";
 import { HashRegistryEntry, PublicKey, SecretKey } from "./crypto";
 import { RegistryEntry, SignedRegistryEntry } from "./registry";
-import { parseSkylink, promiseTimeout, trimUriPrefix, uriSkynetPrefix } from "./utils";
+import { parseSkylink, trimUriPrefix, uriSkynetPrefix } from "./utils";
 
+/**
+ * Gets the JSON object corresponding to the publicKey and dataKey.
+ * @param publicKey - The user public key.
+ * @param dataKey - The key of the data to fetch for the given user.
+ * @param [customOptions={}] - Additional settings that can optionally be set.
+ * @param [customOptions.timeout_seconds=30] - Timeout in seconds for the registry lookup.
```

Comment:
`timeout_seconds` should be `timeout` and accept milliseconds - it's the standard in javascript to use milliseconds (check setTimeout as an example)